### PR TITLE
Extend Bazel build support to bzlmod

### PR DESCRIPTION
**Pull request by Vertexwahn**
Add support for Bazel`s new way of managing dependencies (Bzlmod)

### CODE REVIEW

File: support/bazel/README.md [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+
+The [Bazel Central Registry](https://github.com/bazelbuild/bazel-central-registry/tree/main/modules/fmt) provides support for {fmt}.
+
+For instance, to use {fmt} add to your `MODULE.bazel` file:
+
+```
+bazel_dep(name = "fmt", version = "10.2.1")
+```
+
+For a live at head approach, you can copy the contents of this repository and move the Bazel-related build files to the root folder of this project as described above and make use of `local_path_override`, e.g.:
```
Comment (vitaut) [inline support/bazel/README.md:19]: "live at head" -> "live-at-head" ?

**Review by vitaut — COMMENTED** [review]
LGTM, just one nit inline

File: support/bazel/README.md (hunk 1 above)
Comment (Vertexwahn) [reply support/bazel/README.md]: fixed

**Comment by vitaut** [comment]
Thank you!

**Pull request merged**